types/index.d.ts

```diff
@@ -2605,6 +2605,17 @@ export declare namespace Knex {
     forceFreeMigrationsLock(config?: MigratorConfig): Promise<any>;
   }
 
+  interface Seed {
+    seed: (knex: Knex) => PromiseLike<any>;
+  }
+
+  interface SeedSource<TSeedSpec> {
+    getSeeds(loadExtensions: readonly string[], recursive?: boolean, runSpecificSeed?: boolean): Promise<TSeedSpec[]>;
+    getSeedName(seed: TSeedSpec): string;
```

Comment:
Can't find implementation for FsSeeds

Reply:
What do you mean? `lib/migrations/seed/sources/fs-seeds.js` or something else?

Reply:
Ah, I see, this method does not actually exist.